module/vision/VisualMesh/data/config/networks/RobocupNetwork.yaml

```diff
@@ -0,0 +1,6765 @@
+
```

Comment:
Should we rename this so that it's clear that this shouldn't be overwritten by the webots one? maybe I am overthinking that

Reply:
We can cross that bridge when we actually have more than one trained model